diff --git OptOne.cpp OptOne.cpp
index 0fc2005..281e519 100644
--- OptOne.cpp
+++ OptOne.cpp
@@ -25,12 +25,12 @@ static void task(benchmark::State &state) {
 
   // Original task
   for (auto _ : state) {
-    state.PauseTiming();
+    //state.PauseTiming();
     int NUM = length;
     char flags[8192 + 1];
     long i, k;
     int count = 0;
-    state.ResumeTiming();
+    //state.ResumeTiming();
     while (NUM--) {
       count = 0;
       for (i = 2; i <= 8192; i++) {
@@ -45,9 +45,9 @@ static void task(benchmark::State &state) {
         count = count + (!res);
       }
     }
-    state.PauseTiming();
-    assert(count == 1028);
-    state.ResumeTiming();
+    //state.PauseTiming();
+    //assert(count == 1028);
+    //state.ResumeTiming();
   }
 
   __itt_task_end(domain);
